requirements.d/development.txt

```diff
@@ -11,3 +11,4 @@ pytest-benchmark
 Cython!=0.27
 twine
 python-dateutil
+argon2-cffi
```

Comment:
guess this rather should be in setup.py.

Reply:
Note to self: CI did not fail. It should have.

Reply:
on CI, we have a development/testing setup.

but normal borg users will just have what's pulled via setup.py (or pyproject.toml).